libcontainer/cgroups/fs2/io.go

```diff
@@ -191,3 +191,11 @@ func statIo(dirPath string, stats *cgroups.Stats) error {
 	stats.BlkioStats = parsedStats
 	return nil
 }
+
+func statIoPressure(dirPath string, stats *cgroups.Stats) error {
+	const file = "io.pressure"
+	var err error
+
+	stats.BlkioStats.PSI, err = statPSI(dirPath, file)
+	return err
+}
```

Comment:
I'd rewrite this one to
```go
func statIoPressure(dirPath string, stats *cgroups.Stats) (Err error) {
	stats.BlkioStats.PSI, Err = statPSI(dirPath, "io.pressure")
	return
}
```

Reply:
but with this rewrite, it doesn't make much sense to have it as a separate function, right?